src/modules/displayComments.js

```diff
@@ -0,0 +1,41 @@
+import countComments from './countComments.js';
+
+const displayComments = async () => {
+  const commentId = document.querySelector('.movie-title').id;
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/IR7MovTrVQtBQVyC6UTK/comments?item_id=${commentId}`;
+
+  const commentsList = document.querySelector('.movie-comments');
+  // commentsList.innerHTML = '';
+
+  try {
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    // const commentsCounter = document.querySelector('.comments-counter');
+
+    // if (response.error === 400) {
+    //   commentsCounter.innerHTML = '<p>Comments (0)</p>';
+    // }
+
+    const commentsData = await response.json();
+
+    commentsData.forEach((comment) => {
+      commentsList.innerHTML += `
+        <li>
+          <p class="date">${comment.creation_date}</p>
+          <p class="user-name">${comment.username}:</p>
+          <p class="user-comment">${comment.comment}</p>
+        </li>`;
+    });
+  } catch (error) {
+    // eslint-disable-next-line no-console
```

Comment:
- [ ] For best practice reasons and since it is a common practice in real working environment, I suggest fixing this listing issue instead of disabling linter checks here. Kindly find a way to resolve this without obstructing linter checks :pray: